src/Tools/RepoUtil/ChangeCommand.cs

```diff
@@ -202,7 +202,7 @@ private void ChangeGeneratedFiles()
             var msbuildData = _repoData.RepoConfig.MSBuildGenerateData;
             if (msbuildData.HasValue)
             {
-                var fileName = new FileName(_generateDirectory, msbuildData.Value.RelativeFilePath);
+                var fileName = new FileName(_generateDirectory, msbuildData.Value.RelativeFilePath.Replace("\\", "/"));
```

Comment:
This feels like the wrong place to do the normalization (assuming it is needed)?